<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BerryBoot menu editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QListWidget" name="list">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_OS"/>
   <addaction name="actionEdit"/>
   <addaction name="actionExport"/>
   <addaction name="actionDelete"/>
   <addaction name="actionSet_default"/>
   <addaction name="actionAdvanced_configuration"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="advToolbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionRecover"/>
   <addaction name="actionClone"/>
   <addaction name="actionConsole"/>
   <addaction name="actionSetPassword"/>
   <addaction name="actionRepair_file_system"/>
  </widget>
  <action name="actionAdd_OS">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/install.png</normaloff>:/icons/install.png</iconset>
   </property>
   <property name="text">
    <string>Add OS</string>
   </property>
   <property name="toolTip">
    <string>Install another operating system</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/edit-rename.png</normaloff>:/icons/edit-rename.png</iconset>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="toolTip">
    <string>Edit name or memory split</string>
   </property>
  </action>
  <action name="actionClone">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/editcopy.png</normaloff>:/icons/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>Clone</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSet_default">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/bookmark_large.png</normaloff>:/icons/bookmark_large.png</iconset>
   </property>
   <property name="text">
    <string>Set default</string>
   </property>
   <property name="toolTip">
    <string>Make default</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/door_in.png</normaloff>:/icons/door_in.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/filesave.png</normaloff>:/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Backup</string>
   </property>
  </action>
  <action name="actionAdvanced_configuration">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/configure.png</normaloff>:/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>Edit config</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/commandprompt.png</normaloff>:/icons/commandprompt.png</iconset>
   </property>
   <property name="text">
    <string>Console</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSetPassword">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/encrypted.png</normaloff>:/icons/encrypted.png</iconset>
   </property>
   <property name="text">
    <string>Set password</string>
   </property>
  </action>
  <action name="actionRepair_file_system">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Repair file system</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRecover">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/document_revert.png</normaloff>:/icons/document_revert.png</iconset>
   </property>
   <property name="text">
    <string>Reset OS</string>
   </property>
   <property name="toolTip">
    <string>Restore original image</string>
   </property>
  </action>
  <action name="actionAdvMenu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/moreoptions.png</normaloff>:/icons/moreoptions.png</iconset>
   </property>
   <property name="text">
    <string>More options</string>
   </property>
   <property name="toolTip">
    <string>Display extra buttons</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
